aws/modules/register_group/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "vpc_id" {
 }
 
 variable "instance_ami" {
-  default = "ami-c51e3eb6"
+  default = "ami-98ecb7fe"
```

Comment:
How would you feel about using this?: https://www.terraform.io/docs/providers/aws/d/ami.html